lib/bundler/resolver.rb

```diff
@@ -277,11 +277,11 @@ def name_for(dependency)
     end
 
     def name_for_explicit_dependency_source
-      Bundler.default_gemfile.basename.to_s rescue "Gemfile"
+      "#{Bundler.default_gemfile.basename}" rescue "gems.rb"
```

Comment:
because interpolation implies there's more to the string than just the bit your interpolating, and `to_s` is clearer about what's happening here